.github/workflows/pr-release.yml

```diff
@@ -248,20 +314,20 @@ jobs:
 
           echo "Using base tag: $BASE"
 
-          git checkout "$BASE"
-
           EXISTS="$(git ls-remote --heads origin lean-pr-testing-${{ steps.workflow-info.outputs.pullRequestNumber }} | wc -l)"
           echo "Branch exists: $EXISTS"
           if [ "$EXISTS" = "0" ]; then
             echo "Branch does not exist, creating it."
-            git checkout -b lean-pr-testing-${{ steps.workflow-info.outputs.pullRequestNumber }}
+            git switch -c lean-pr-testing-${{ steps.workflow-info.outputs.pullRequestNumber }} "$BASE"
```

Comment:
Hmm turns out
```
git switch -c foo base
```
works if `base` is a local tag, but not if it is a (remote) branch.

The old code (`git checkout base`) would work for a (remote) branch, but not for a tag. Kinda annoying.

Reply:
Maybe we need to parse this output to go onto a common path in both cases

```
$ git ls-remote --heads --tags --exit-code https://github.com/leanprover-community/mathlib4 nightly-testing-2024-01-22
7d803278d1b5978fccc14f992d9ffb692077ed93	refs/heads/nightly-testing-2024-01-22
$ git ls-remote --heads --tags --exit-code https://github.com/leanprover/std4 nightly-testing-2024-01-22
c15b5a8f04e84bc7aab8b1062a4a80de3046865f	refs/tags/nightly-testing-2024-01-22
```

Alternatively, we can once manually convert all `nightly-testing-YYYY-MM-DD` branches to tags on mathlib4. Maybe easier.

Reply:
Possible fix in https://github.com/leanprover/lean4/pull/3208